sdk/search/Azure.Search.Documents/README.md

```diff
@@ -47,9 +47,9 @@ dotnet add package Azure.Search.Documents --version 1.0.0-preview.2
 ### Prerequisites
 
 You need an [Azure subscription][azure_sub] and a
-[Search Service][create_search_service_docs] to use this package.
+[search service][create_search_service_docs] to use this package.
```

Comment:
Any reason we stopped capitalizing "Search"? "service" shouldn't be since it's not a proper name, but in Key Vault we just went through a branding/doc exercise to uniformly refer to "vaults" as "Azure Key Vaults", etc. In fact, probably should change "search" (when referring to the service name) to "Azure Search". Going to have to do it sooner or later, I imagine.

Reply:
I included a bunch of @HeidiSteen's proposed fixes from https://github.com/HeidiSteen/azure-sdk-for-net/pull/3/files after the last doc review.  Except the double spaces, obviously.

Reply:
We could probably resolve this for preview 4, but, @HeidiSteen, since "Search" is a product name shouldn't this be "Search" or even "Azure Search" (or maybe "Azure Cognitive Search") per branding guidelines (I had to do quite a bit of these changes back in the day for .NET and Visual Studio, including on my formerly popular blog - they were pretty serious).

Reply:
I'm going merge the PR now to avoid any conflicts, but we can continue the discussion about the README.

Reply:
Agreed, as noted. I linked a new issue for preview 4.